src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -130,6 +133,57 @@ public void should_attach_exception_record_to_the_existing_case_with_evidence_do
         verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
     }
 
+    @Test
+    public void should_attach_supplementary_evidence_with_pending_payments() throws Exception {
+        //given
+        CaseDetails caseDetails = ccdCaseCreator.createCase(emptyList(), now()); // with no scanned documents
+        CaseDetails exceptionRecord = createExceptionRecord(
+            "envelopes/supplementary-evidence-envelope-with-payment.json"
+        );
+
+        // when
+        attachExceptionRecord(caseDetails, exceptionRecord, null);
+
+        //then
+        await("Exception record with pending payments is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(caseDetails, 1));
```

Comment:
Is this wait needed?

Reply:
according to the config in #860, supplementary_evidence is allowed with pending payments

Reply:
we have added wait for all the existing tests

Reply:
Because we're calling the service directly from tests and everything happens synchronously, I would expect the case and exception record to be modified at this point. I don't think this waiting is necessary.